client/layout/utils.ts

```diff
@@ -49,10 +49,14 @@ export const handleScroll = ( event: React.UIEvent< HTMLElement > ): void => {
 
 		if (
 			windowHeight >= secondaryElHeight + masterbarHeight &&
-			content.style.minHeight !== 'initial'
+			secondaryEl !== undefined &&
+			secondaryEl !== null &&
```

Comment:
According to the MDN Web Docs, if the element is not found, `getElementById` returns `null` so I think we can simply check that. It could be a good idea to check `content` to, as Tom suggested.

https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById#return_value
```suggestion
			content !== null &&
			secondaryEl !== null &&
```